src/cli/hooks/InstallLwcServerPlugin.ts

```diff
@@ -0,0 +1,24 @@
+import { Logger } from '@salesforce/core';
+import { CommonUtils } from '../../common/CommonUtils';
+
+const logger: Logger = new Logger('force:lightning:local:setup');
+export const hook = async (options: { id: string }) => {
+    const command = 'sfdx plugins:install @salesforce/lwc-dev-server';
+    try {
+        await CommonUtils.isLwcServerPluginInstalled();
```

Comment:
It feels like the LWC plugin functionality should be encapsulated in one class.  Any reason not to control that functionality from one place?

Reply:
This is the first requirement that spans across platforms. Merely establishing a place where they could reside.

Reply:
I agree with a place for them to reside, and think this class/file is a fine place for that.  It's not clear to me why we wouldn't also host `CommonUtils.isLwcServerPluginInstalled()` here too.  What is the value of it residing in `CommonUtils`?

Reply:
@khawkins My intention was to keep most of the code outside the hook itself. e.g. Its getting used by Setup for both platforms as well in-order to validate the requirement. I could just as easily rename CommonUtils to LWCServerUtils or something else :)

Reply:
Oh yeah that makes sense—I forgot this was the actual hook. I was just hoping to consolidate LWC server business functions into a single owner, even it had multiple consumers.  I'm cool with the separation, since it's pretty minor. If we have a need to do more work with the plugin in the future, we may want to reconsider.